tutor/specs/components/task-plan/homework/__snapshots__/add-exercises.spec.cjsx.snap

```diff
@@ -81,8 +81,18 @@ exports[`choose exercises component matches snapshot 1`] = `
             />
           </div>
           <span>
-            Tutor Selections
+            OpenStax Tutor Selections
           </span>
+          <button
+            aria-describedby="homework-selections-popover"
+            className="btn btn-link"
+            disabled={false}
+            id="homework-selections-trigger"
+            onClick={[Function]}
+            type="button"
+          >
+            What are these?
+          </button>
         </div>
         <div
```

Comment:
im working off this mockup: http://2jsgoo.axshare.com/homework_assignment_builder_tutor_help.html.   the vertical bars were not filling the height even before i added this, it's position absolute so it's not making the box bigger.  i took some artistic license to this -- namely, i did not make the whole bar and vertical lines bigger, and i did not put the `What are this?` to the left.